### ISSUE TITLE
No Validation Set?

### ISSUE THREAD

**Issue by skuley** [plemeri/InSPyReNet#15]
Hi, I am very impressed by idea of your work!

I have a question tho..
Usually in Training supervised model, we tend to validate at the end of each epoch.

Can you describe why validation step has not been include in this work?

Thanks!

**Comment by plemeri** [comment]
Hi, thank you for your attention.

First, we do not have any validation set for the salient object detection benchmarks.
Also, because previous works did not utilize any of validation set, we also do not use any of them. Otherwise, it would be a violation and end up with unfair comparison.

We can actually choose to use a subset of training dataset as a validation dataset for personal use or other application.
I guess what you are trying to do is early stopping in order to get the best checkpoint and I think it would be good idea to use for the production level, but please do not use early stopping or validation set for academic purpose since again, previous works did not use them.

Thank you

**Comment by skuley** [comment]
Thank you for your answer!
